specification/healthcareapis/resource-manager/readme.md

```diff
@@ -28,19 +28,28 @@ These are the global settings for HealthcareApis service.
 title: HealthcareApisManagementClient
 description: Azure Healthcare APIs Client
 openapi-type: arm
-tag: package-preview-2021-06
+tag: package-2021-01
```

Comment:
Shall it be `package-2021-11`?

Reply:
This controls the default version of the API correct? If so this is intentional.  The API verison 2021-11-01 hasn't been published in ARM yet so we can't make this the default version.